app/jobs/trs_mentor_sync_job.rb

```diff
@@ -0,0 +1,11 @@
+class TRSMentorSyncJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Mentor.find_in_batches(batch_size: 200).with_index do |mentors, index|
+      mentors.each do |mentor|
+        TRSUpdateMentorDetailsJob.set(wait: index.minutes).perform_later(mentor)
```

Comment:
Wouldn't it be enough to wait `index.seconds`? That would mean we would never do more than 60 per minute, and the rate limit is 300, right?

Reply:
resolved in [7537d17](https://github.com/DFE-Digital/itt-mentor-services/pull/302/commits/7537d17dc7d8cfa052c688cc1a228c0b8832e82f)